ngraph/frontend/onnx_import/src/op/non_max_suppression.cpp

```diff
@@ -86,11 +86,10 @@ namespace ngraph
 
                     const auto box_encoding =
                         center_point_box == 0
-                            ? ngraph::op::v4::NonMaxSuppression::BoxEncodingType::CORNER
-                            : ngraph::op::v4::NonMaxSuppression::BoxEncodingType::CENTER;
+                            ? default_opset::NonMaxSuppression::BoxEncodingType::CORNER
```

Comment:
Is it ok for all IE plugins?

Reply:
Well, for now we have a situation when generating both NMS-4 or NMS-5 is not ok for all plugins. The reason for this is that the NMS-5 operation was merged separately from the appropriate transformations (I mean this PR https://github.com/openvinotoolkit/openvino/pull/2450). As MyriadPlugin has its own pipeline for NMS operation, we doesn't depend on the transformations above so we have changed the plugin to work with NMS-5, as some models for other plugins were suppressed. I just don't see a reason for the previous change that was made to generate NMS-4 instead of NMS-5 just to temporarily enable some tests on CPU (Anyway we will get this changes back), but it instead it broke some models for VPU.

Reply:
I talked with the ONNX team and in our opinion we can live with this xfail for now, but please mark it with the number of existing ticket or create a new one for it.

PR #2450 should be merged as fast as possible IMO.

Reply:
#2450 was merged.

Reply:
The related transformations were merged, so now it OK for all plugins, but for some reason one of the model is still failing on CPU, I've created a separate ticket on this failure - 42779.